src/components/notifications/HostedSMS.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <div class="mb-3">
+        <label for="hostedsms-username" class="form-label">{{ $t('hostedsmsAPIUser') }}</label>
+        <input id="hostedsms-username" v-model="$parent.notification.hostedsmsUsername" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="hostedsms-key" class="form-label">{{ $t('hostedsmsAPIPassword') }}</label>
+        <HiddenInput id="hostedsms-key" v-model="$parent.notification.hostedsmsPassword" :required="true" autocomplete="new-password"></HiddenInput>
```

Comment:
Not a strong opinion, but why did you add the `autocomplete` here? 
Is this not a key generated by them?

Reply:
This code was based on https://github.com/louislam/uptime-kuma/blob/master/src/components/notifications/SerwerSMS.vue
I removed autocomplete. This is not generated key, but it shouldn't be in browser autocomplete.